PRINCIPLES.md

```diff
@@ -10,20 +10,20 @@ These principles were derived from modern software operations but are rooted in
 
 ## Principles
 
-1. **The principle of declarative desired state**
+1. **Define state declaratively**
 
     A system managed by GitOps must have its _Desired State_ expressed declaratively as data in a format writable and readable by both humans and machines.
 
-2. **The principle of immutable desired state versions**
+2. **Use immutable, versioned storage**
```

Comment:
What about "Use immutable, versioned declaration" (to map to principle 4)

Reply:
Maybe "Store declarations in an immutable version control system"

It's long but I think the weakness of what I have in the PR is it sounds almost like a recommendation about what kind of storage your running infra should use which doesn't make sense. 

This rephrasing feels like it emphasizes the declaration.